src/plugins/saved_search/public/services/saved_searches/get_saved_searches.ts

```diff
@@ -20,6 +21,7 @@ interface GetSavedSearchDependencies {
   search: DataPublicPluginStart['search'];
   savedObjectsClient: SavedObjectsStart['client'];
```

Comment:
NIT: (outside of the scope of the PR) `savedObjectsClient: SavedObjectsClientContract`